### PR TITLE
Move OpenApiSpecBuilder to a standalone package

### PR DESCRIPTION
**Pull request by bajtos**
Connect to #135

cc @bajtos @raymondfeng @ritch @superkhau

### CODE REVIEW

File: packages/openapi-spec-builder/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# @loopback/openapi-spec-builder
+
+Builder API making it easy to create OpenAPI Spec/Swagger documents in your tests
```
Comment (ritch) [inline packages/openapi-spec-builder/README.md]: This sentence doesn't read right...?

Comment (superkhau) [inline packages/openapi-spec-builder/README.md]: Making it easy to create OpenAPI (Swagger) specification documents in your tests using the builder pattern.

File: packages/openapi-spec-builder/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+shared swagger boilerplate.
+
+OpenApiSpecBuilder utilizes
+[Test Data Builder pattern](http://www.natpryce.com/articles/000714.html)
+to provide a TypeScript/JavaScript API allowing users to create
+full OpenAPI Spec documents in few lines of code.
+
+## Installation
+
+```sh
```
Comment (superkhau) [inline packages/openapi-spec-builder/README.md]: sh = shell?

File: packages/openapi-spec-builder/package.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@loopback/openapi-spec-builder",
+  "version": "4.0.0-alpha.1",
+  "description": "Builder API making it easy to create OpenAPI Spec/Swagger documents in your tests",
```
Comment (superkhau) [inline packages/openapi-spec-builder/package.json]: Making it easy to create OpenAPI (Swagger) specification documents in your tests using the builder pattern.

**Pull request merged**